datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/ingest/IngestionResolverUtils.java

```diff
@@ -83,7 +85,11 @@ public static com.linkedin.datahub.graphql.generated.ExecutionRequestResult mapE
   public static List<IngestionSource> mapIngestionSources(final Collection<EntityResponse> entities) {
     final List<IngestionSource> results = new ArrayList<>();
     for (EntityResponse response : entities) {
-      results.add(mapIngestionSource(response));
+      try {
+        results.add(mapIngestionSource(response));
+      } catch (IllegalStateException e) {
+        log.error("Unable to map ingestion source, continuing to other sources.", e);
```

Comment:
So here we silently skip when there's a malformed ingestion source?

Reply:
Throwing an error here results in the page not loading at all, this gives a record in the logs of it occurring and lets the non-malformed go through. Since malformed ingestion sources are not expected except in weird situations, this allows normal operations to continye while allowing the malformed source to be debugged.

If we need the error to actually get propagated in the response I can think a bit more on how to go about that without hard erroring, but I think this is a better experience for an unexpected situation.

Reply:
Okay got it lets go with it